src/transactions/feed/queryHelper.ts

```diff
@@ -357,9 +357,10 @@ async function queryChainTransactionsFeed({
       ...body.data,
       tokenTransactionsV3: {
         ...body.data.tokenTransactionsV3,
-        transactions: body.data.tokenTransactionsV3.transactions.map((tx) => {
-          return { ...tx, networkId }
-        }),
+        transactions:
+          body.data.tokenTransactionsV3?.transactions.map((tx) => {
```

Comment:
we have optional chaining on `tokenTransactionsV3` in other parts of this file already, but this part was causing the homefeed to break when i was testing with a user who had only Ethereum transactions (`tokenTransactionsV3` was returned as null for Celo requests)